src/index.js

```diff
@@ -16,9 +17,11 @@ import {
  *
  * @param {string} selector
  * @param {Function} eventFunctions
+ * @param {string} server
  * @return {Promise<Array<ExecutableCode>>}
  */
-export default function create(selector, eventFunctions) {
+export default function create(selector, eventFunctions, server) {
```

Comment:
I'd prefer a mechanism that allows any arbitrary feature like event, server. rather than adding more and more arguments to functions.
It can be `options` argument instead of `eventFunctions`. What do you think about it?

Reply:
Yeah, it would definitely be a better approach!  Let me tweak it :+1: